### PR TITLE
Speed change

### PR DESCRIPTION
**Pull request by apokluda**
Just making the game a _little_ bit more fun ⏩

### CODE REVIEW

**Comment by apokluda** [comment]
@lost-grail What's the hold up?

**Comment by lost-grail** [comment]
@apokluda this change requires an architecture review

File: index.html [hunk 1]
```diff
@@ -75,7 +75,7 @@
         ctx     = canvas.getContext('2d'),
         ucanvas = get('upcoming'),
         uctx    = ucanvas.getContext('2d'),
-        speed   = { start: 0.6, decrement: 0.005, min: 0.1 }, // how long before piece drops by 1 row (seconds)
+        speed   = { start: 6000, decrement: 0.5, min: 0.1 }, // how long before piece drops by 1 row (seconds)
```
Comment (lost-grail) [inline index.html]: How slow do you think I am?

Comment (apokluda) [reply index.html]: 6000?

Comment (lost-grail) [reply index.html]: that is, about 100 minutes for the block to fall one step

Comment (lost-grail) [reply index.html]: also, not sure how to get the github/pages status check to pass

Comment (apokluda) [reply index.html]: @beardofedu Any help?

Comment (apokluda) [reply index.html]: @beardofedu @lost-grail Maybe you need to approve before the check will run?

**Pull request merged**